app/assets/scss/application.scss

```diff
@@ -48,35 +34,49 @@ $path: "/suppliers/static/images/";
 @import "toolkit/_secondary-action-link.scss";
 @import "toolkit/_service-id.scss";
 @import "toolkit/_temporary-message.scss";
+@import "toolkit/forms/_combinations.scss";
+@import "toolkit/forms/_checkbox-tree.scss";  // Has to be imported after _combinations.scss, which defines the
+                                              // $border-width variable that _checkbox-tree.scss relies upon. :sad:
```

Comment:
Well that's a shame. Good spot though. I think we shouldn't have one of our partials depending on a variable set in another partial. _checkbox-tree.scss should likely be defining it's own $border-width variable if it needs one (or if it is truely a shared variable then should be stored in a different file and imported in by partials that need it). If you agree, could you write a ticket at the bottom of our icebox for this please :)

Reply:
Yep, ticket made.